Updated images on docs for authenticate azure and abbout connections (#6391)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/connect-data-platform/about-connections.md

- [Starburst or Trino](/docs/cloud/connect-data-platform/connect-starburst-trino)
- [Teradata](/docs/cloud/connect-data-platform/connect-teradata) <Lifecycle status="Preview" />

To connect to your database in dbt Cloud:

1. Click your account name at the bottom of the left-side menu and click **Account settings**

> 3. Select **Projects** from the top left, and from there click **New Project**

Author: ```suggestion
2. Select **Projects** from the top left, and from there click **New Project**
```

- [Starburst or Trino](/docs/cloud/connect-data-platform/connect-starburst-trino)
- [Teradata](/docs/cloud/connect-data-platform/connect-teradata) <Lifecycle status="Preview" />

> You can connect to your database in dbt Cloud by:

Reviewer: You can leave this as is, but the following instructions would be "clicking" if you read right after. It might flow a little better like this
```suggestion
To connect to your database in dbt Cloud:
```

> 1. Click your name at the bottom left of your account

Reviewer: ```suggestion
1. Click your name at the bottom of the menu bar
```

> 1. Click your name at the bottom left of your account

Reviewer: The way you've written it here is totally valid and 100% a way that a user can navigate to the settings, but it might be beneficial to the user to lead them there via the Account settings workflow
```suggestion
1. Click your account name at the bottom of the left-side menu and click **Account settings**
```

> 2. Select **Your profile** from the menu

Reviewer: If you go with the above, you can delete step 2 here

> 2. Select **Your profile** from the menu

Author: ```suggestion

```

> 3. Select **Projects** from the top left, and from there click **+ New Project**

Reviewer: You can use just the text, you don't have to include symbols
```suggestion
3. Select **Projects** from the top left, and from there click **New Project**
```
